src/components/EditOrganization/index.tsx

```diff
@@ -132,6 +133,27 @@ export default function OrganizationView({ organization, reloadOrganizationData
     return 6;
   };
 
+  const countryToDropdownItem = (country: Country) => {
+    return { label: country.name, value: country.code };
+  };
+
+  const countriesOptions = () => {
+    if (countries) {
+      return countries.map((country) => countryToDropdownItem(country));
+    }
+  };
```

Comment:
return empty array otherwise.. we can remove the || [] from the autocomplete..
might be good for these functions to return the options based on current state of the component.. not only if countries or country is set?